vllm/utils/tensor_schema.py

```diff
@@ -0,0 +1,188 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+
+from typing import Annotated, Any, Union, get_args, get_origin, get_type_hints
+
+import torch
+
+
+class TensorShape:
+
+    def __init__(self,
+                 *dims: Union[int, str],
+                 dynamic_dims: set[str] = None) -> None:
+        self.dims = dims
+        self.dynamic_dims = dynamic_dims if dynamic_dims else set()
+
+    def resolve(self, **bindings: dict[str, int]) -> tuple[Union[str, int]]:
```

Comment:
```suggestion
    def resolve(self, **bindings: dict[str, int]) -> tuple[Union[str, int], ...]:
```

For tuples with any number of elements, you need to add ellipsis to the end

Reply:
Thanks! Reflected in latest update.